src/test/scala/edu/berkeley/cs/rise/opaque/TPCHTests.scala

```diff
@@ -26,94 +26,94 @@ import edu.berkeley.cs.rise.opaque.benchmark.TPCH
 trait TPCHTests extends OpaqueTestsBase { self => 
 
   def size = "sf_small"
-  def tpch = TPCH(spark.sqlContext, size)
+  def tpch = new TPCH(spark.sqlContext, size, "file://")
 
   testAgainstSpark("TPC-H 1") { securityLevel =>
-    tpch.query(1, securityLevel, spark.sqlContext, numPartitions).collect
+    tpch.query(1, securityLevel, numPartitions).collect
   }
```

Comment:
Maybe this can be simplified to have a function make a call repeated to testAgainstSpark for these query numbers?